db/monitor/ACR06B1.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<monitor name="Acer Nitro XV273K" init="standard">
+	<caps add="(prot(monitor)type(lcd)model(XV273K)cmds(01 02 03 07 0C E3 F3)vcp(02 04 05 08 0B 0C 10 12 14(05 06 08 0B 0C) 16 18 1A 52 54(00 01) 59 5A 5B 5C 5D 5E 60(03 11 0F 10) 62 9B 9C 9D 9E 9F A0 AC AE B6 C0 C6 C8 C9 CC(01 02 03 04 05 06 08 09 0A 0C 0D 0E 14 16 1E)D6(01 04 05) DF E3 E5 E6 E7(00 01 02) E8(00 01 02) )mswhql(1)asset_eep(32)mccs_ver(2.1))" />
+
+	<controls>
+		<control id="brightness" address="0x10"/>
+		<control id="contrast" address="0x12"/>
+		<control id="red" address="0x16"/>
+		<control id="green" address="0x18"/>
+		<control id="blue" address="0x1A"/>
```

Comment:
```suggestion
```

Please, don't duplicate controls, which are identical to ones, which are in vesa profile.

Ideally, these profiles should contain only vendor/model specific controls. However, due to current ddccontrol's limitations, controls with options show all options, regardless of caps, (i.e. limit to `60(03 11 0F 10)`).

Add include tag at the end of the profile:

```
	<include file="VESA"/>
```


I.e. https://github.com/ddccontrol/ddccontrol-db/blob/master/db/monitor/DEL4015.xml

Reply:
Fixed!